cfme/containers/provider/__init__.py

```diff
@@ -149,6 +149,15 @@ def summary_text(self):
         return self.SUMMARY_TEXT
 
 
+class ContainersProviderDashboardView(ContainerProviderDetailsView):
+
+    @property
+    def is_displayed(self):
+        return(
```

Comment:
REQUIRED: typo ?

Reply:
where do you see the typo?

Reply:
there should be at least some space between return and (